Pair-Routes/about.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div>This is the aboutHTML</div>
```

Comment:
You can leave a space between `about` and `HTML`

Reply:
Oh thats an amazing suggestion TY!